src/app/home/pages/PlanDebugPage/helpers.ts

```diff
@@ -38,97 +38,137 @@ export const getFullKindName = (kind: any) => {
+    case 'Route':
+      return false;
+    default:
+      return kind;
+  }
+};
+
 interface ICommandAndTypeObj {
-  ocCommand: string;
-  clusterType: string;
+  ocDescribeCommand: string;
+  ocLogsCommand: string;
 }
-export const getOCCommandAndClusterType = (rawNode: IDebugTreeNode): ICommandAndTypeObj => {
+export const getDebugCommand = (rawNode: IDebugTreeNode): ICommandAndTypeObj => {
   const { kind, namespace, name } = rawNode;
   switch (kind) {
     case 'DirectImageStreamMigration':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: `oc logs --selector control-plane=controller-manager -n ${namespace} --container mtc | grep '"dism":"${name}"'`,
       };
     case 'DirectVolumeMigrationProgress':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: `oc logs --selector control-plane=controller-manager -n ${namespace} --container mtc | grep '"dvmp":"${name}"'`,
       };
     case 'Hook':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: `oc logs --selector control-plane=controller-manager -n ${namespace} --container mtc | grep '"migHook":"${name}"'`,
       };
     case 'Job':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: null,
       };
     case 'PV':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} ${name}`,
+        ocLogsCommand: null,
       };
     case 'PVC':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: null,
       };
     case 'Route':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: null,
       };
     case 'Pod':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: `oc logs -n ${namespace} ${name}`,
```

Comment:
Does this one need a container specified also?

Was seeing this output when trying to run the oc logs command 
``` oc logs -n mediawiki dvm-rsync-vdmpn
error: a container name must be specified for pod dvm-rsync-vdmpn, choose one of: [rsync-client stunnel]
```

Reply:
Resolved in latest commit 👍 , I also made `-n` longform as `--namespace` in all generated commands so the user can have a better chance of understanding what's going on.